src/mui/detail/RecordForm.js

```diff
@@ -0,0 +1,80 @@
+import React, { PropTypes } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { Toolbar, ToolbarGroup } from 'material-ui/Toolbar';
+import { SaveButton } from '../button';
+import { Labeled } from '../input';
+
+let validators = {};
+
+const validate = (values) => {
```

Comment:
A solution is to move validation to the `<Create />` or `<Edit />` component, something like:

``` js
<Edit title={PostTitle} validation={{
        title: { required: true },
        teaser: { required: true },
        average_note: { min: 0, max: 5 },
        body: { custom: (value, record) => {
            if (value === record.teaser) {
                return 'Body and teaser should be different.';
            }

            return '';
        }}
    }} {...props}>
```
What do you think of this format?